### PR TITLE
Dev improvements

### PR DESCRIPTION
**Pull request by raq929**
Adds PR template
Updates the Makefile help command
Ignores pyenv version files

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+**Post merge:**
+- [ ] Tag pushed and a new [version](https://github.com/edx/ecommerce-worker/releases) released
+    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
+- [ ] After versioned build finishes in [Travis](https://travis-ci.org/github/edx/ecommerce-worker), verify version has been pushed to [PyPI](https://pypi.org/project/edx-ecommerce-worker/)
+    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
+    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
+- [ ] PR created in [ecommerce](https://github.com/edx/ecommerce) to upgrade dependencies (including ecommerce-worker)
+    - This **must** be done after the version is visible in PyPi as `make upgrade` in ecommerce will look for the latest version in PyPi.
+    - Note: the ecommerce-worker constraint in ecommerce **must** also be bumped to the latest version in PyPi.
+- [ ] Deploy `ecommerce`
```
Comment (binodpant) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: any links available to help with these?

**Review by binodpant — APPROVED** [review]
other than a question about possibly adding more links, this lgtm! previewed the links etc and pr template at https://github.com/edx/ecommerce-worker/blob/48e172b02a74c5b24873e6c21b84b51ac6f10d9b/.github/PULL_REQUEST_TEMPLATE.md and looks fine to me!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (raq929) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: I'd just be linking to the goCD dashboard; searching in GoCD doesn't update the URL in a useful way, so I don't think it's worth it?

Comment (binodpant) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: ah ok np

**Pull request merged**